### PR TITLE
Update list of solaris ethernet drivers

### LINKED ISSUE #549: no macaddress on solaris
Dear developers,

I recognized, that ohai does not find any macaddress on solaris 10 while an ipaddress is found. I'm using Chef client 11.14.6 and ohai 7.2.4. There's no difference between sparc and x86.

here are some outputs:
 # ohai
{ ...
  "ipaddress": "192.168.3.160",
  "macaddress": null,
  ...
}

 # ohai macaddress
/opt/chef/embedded/lib/ruby/gems/1.9.1/gems/ohai-7.2.4/lib/ohai/system.rb:191:in `attributes_print': I cannot find an attribute named macaddress! (ArgumentError)
        from /opt/chef/embedded/lib/ruby/gems/1.9.1/gems/ohai-7.2.4/lib/ohai/application.rb:94:in `block in run_application'
        from /opt/chef/embedded/lib/ruby/gems/1.9.1/gems/ohai-7.2.4/lib/ohai/application.rb:93:in `each'
        from /opt/chef/embedded/lib/ruby/gems/1.9.1/gems/ohai-7.2.4/lib/ohai/application.rb:93:in `run_application'
        from /opt/chef/embedded/lib/ruby/gems/1.9.1/gems/ohai-7.2.4/lib/ohai/application.rb:70:in `run'
        from /opt/chef/embedded/lib/ruby/gems/1.9.1/gems/ohai-7.2.4/bin/ohai:42:in `<top (required)>'
        from /usr/bin/ohai:23:in `load'
        from /usr/bin/ohai:23:in `<main>'

Do you have an idea what's going wrong?

Kind regards,
Lothar

### PR DESCRIPTION
**Pull request by thommay**
We're failing to detect non intel ethernet drivers, which means that on some solaris nodes we don't get a mac address or a default IP. 
Closes #549

### CODE REVIEW

**Comment by thommay** [comment]
The travis failure is the json utf8 weirdness again.

**Comment by thommay** [comment]
@lhandl if you want to try this with chef 12:
```
curl -O https://raw.githubusercontent.com/chef/ohai/tm/solaris_mac/lib/ohai/plugins/solaris2/network.rb
pfexec cp network.rb /opt/chef/embedded/apps/ohai/lib/ohai/plugins/solaris2/network.rb
```
and let me know if you get a reported macaddr please?

**Comment by lhandl** [comment]
works perfectly. Thank you very much.

**Comment by thommay** [comment]
@chef/client-core review please?

**Comment by tyler-ball** [comment]
:+1: purely to the code - I dunno' nuthin' about Solaris

**Comment by lamont-granquist** [comment]
Travis failure should go away if you rebase on master.

:+1: on the code.

File: lib/ohai/plugins/solaris2/network.rb [hunk 1]
```diff
@@ -119,7 +125,7 @@ def arpname_to_ifname(iface, arpname)
 
     so = shell_out("arp -an")
     so.stdout.lines do |line|
-      if line =~ /([0-9a-zA-Z]+)\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\w+)\s+([a-zA-Z0-9\.\:\-]+)/
+      if line =~ /([0-9a-zA-Z]+)\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\w+)?\s+([a-zA-Z0-9\.\:\-]+)/
```
Comment (danielsdeleo) [inline lib/ohai/plugins/solaris2/network.rb]: If you have an example of what this line matches handy, it'd be :100: to have a comment about it. Long regex is long.

**Comment by danielsdeleo** [comment]
:+1: 

Also we should configure travis to only build the PR and not the branch.

**Pull request merged**